Makefile

```diff
@@ -34,12 +34,14 @@ EXTRA_LD_FLAGS =
 LD_FLAGS = $(BUILTIN_LD_FLAGS) $(EXTRA_LD_FLAGS)
 
 # BUILTIN_GC_FLAGS are the internal flags used to pass compiler.
-BUILTIN_GC_FLAGS =
+BUILTIN_GC_FLAGS = all=-trimpath="$$HOME"
```

Comment:
Should there be two `$` here?

Reply:
Yeah, that's intentional, since in a Makefile, `$` is used to specify make variables. If you want a literal `$` so it can be interpreted by the shell, you need to escape it, so `$$`. So what the shell sees is `all=-trimpath="$HOME"` and it does the right substitution.

We want this to be interpreted by the shell, since on Windows it's possible that we may end up with backslashes, and it's better to let the shell handle all of that, since writing `$(HOME)` and letting the Makefile do the substitution could result in odd behavior with backslashes and such.

I probably should have put all of this in the commit message, come to think of it.

Reply:
I've updated the commit message and made these values overridable for cases when users would rather user `PWD` instead of `HOME`.